docs/assets/C#NamingGuidelinesTable.md

```diff
@@ -0,0 +1,15 @@
+# C# Naming Conventions - Quick Reference Table
+
+| Kind                                 | Naming Convention |  Example                                                           |
+| ------------------------------------ | ----------------- | ------------------------------------------------------------------ |
+| Classes                              | PascalCase        | `class Car{}`                                                      |
+| Types and Namespaces                 | PascalCase        | `namespace VehicleManufacturer{}`                                  |
+| Parameters                           | camelCase         | `public Car(int odometerMileage, string manufacturer)`             |
+| Methods                              | PascalCase        | `public void StartEngine()`                                        |
+| Properties                           | PascalCase        | `public double FuelLevel { get; set; }`                            |
+| Local Variables                      | camelCase         | `public int yearManufactured;`                                     |
+| Local Functions                      | PascalCase        | `private string CalculateMilesUntilEmpty(double fuelLevel)`        |
+| Fields                               | _PascalCase       | `public string _Day;`                                              |
```

Comment:
I would propose that the access modifiers are not adding much to these examples (esp since we don't have rules that differ based on them). Would propose removing them on all items. Thoughts?
```suggestion
| Fields                               | _PascalCase       | `string _Day;`                                              |
```

Reply:
I guess the biggest advantage to not removing them right now would be to differentiate from local variables, functions that don't have access modifiers to show examples that should have them vs not? As well as with keeping it for a beginner it is more closely following an actual code snippet which is maybe helpful for beginners? But that would be my only reasons for not removing them all.

Reply:
I think this also ties with your other comment with the {}. How much do we want the examples to explicitly show just the naming convention, or allow it to also be an example of a code snippet of that specific kind?

Reply:
That is fine leaving them in, though I would then propose that this one should be `private` not `public`